src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend.py

```diff
@@ -49,18 +49,21 @@
 
 openvino_options = {}
 
-@register_backend
+
 @fake_tensor_unsupported
 def openvino(subgraph, example_inputs, options=None):
-    if (_get_aot_autograd(options)):
+    if _get_aot_autograd(options):
         global openvino_options
         openvino_options = options
         decompositions = _get_decompositions(options) + get_inf_decomposition_list() + get_aot_decomposition_list()
-        return aot_autograd(fw_compiler=fx_openvino,
-                            bw_compiler=fx_openvino,
-                            decompositions=get_decompositions(decompositions))(subgraph, example_inputs)
+        return aot_autograd(fw_compiler=fx_openvino, bw_compiler=fx_openvino, decompositions=get_decompositions(decompositions))(subgraph, example_inputs)
     return fx_openvino(subgraph, example_inputs, options)
 
+
+if torch.__version__ < "2.5.0":
```

Comment:
This is not reliable (consider version 2.10.0 of torch if released), please fix this using `version` from `packaging`.

Reply:
Done